### PR TITLE
feat(combobox): match combobox height with single selection mode

### PR DESCRIPTION
**Pull request by anveshmekala**
**Related Issue:** #2624 

## Summary

Match `combobox` height when selection mode is `multi `or `ancestor` with single-selection mode.

### CODE REVIEW

**Comment by macandcheese** [comment]
Looks good, thanks @anveshmekala

File: src/components/calcite-combobox/calcite-combobox.tsx [hunk 1]
```diff
@@ -952,8 +951,8 @@ export class CalciteCombobox {
           aria-owns={guid}
           class={{
             wrapper: true,
-            "wrapper--active": open,
-            "wrapper--single": single
+            "wrapper--single": single || !this.selectedItems.length,
```
Comment (caripizza) [inline src/components/calcite-combobox/calcite-combobox.tsx:954]: Can you revert the other file changes if this is the only fix needed?

**Review by caripizza — APPROVED** [review]
Looks good to me 👍  I asked Adam about the CSS transition in the issue, let's get his response before merging. Also don't forget to add your username in front of the branch name [per convention](https://github.com/Esri/calcite-components/blob/master/CONTRIBUTING.md#branch-naming-conventions).

File: src/components/calcite-combobox/calcite-combobox.tsx (hunk 1 above)
Comment (anveshmekala) [reply src/components/calcite-combobox/calcite-combobox.tsx:954]: there is one more line of code that's changed [:858](https://github.com/Esri/calcite-components/pull/3094/files#diff-3c5e42fd5fba0b231584840e4808faf46b8f2cdbda50c5b3bdd1b86dfa779378R858).

**Pull request merged**